src/apps/testapps/mkRandGeoBoundary.c

```diff
@@ -64,11 +64,11 @@ int main(int argc, char* argv[]) {
     }
 
     for (int i = 0; i < numPoints; i++) {
-        GeoPoint g;
+        LatLng g;
```

Comment:
Nit: Not really worth fixing everywhere, but `g` is now a completely confusing short var name for `LatLng` - ideally these would be `latLng` or `ll`.

Reply:
Moving discussion out to https://github.com/uber/h3/issues/483